reference/twig_reference.rst

```diff
@@ -95,7 +95,7 @@ controller
     **type**: ``array`` **default**: ``[]``
 
 Returns an instance of ``ControllerReference`` to be used with functions like
-:ref:`render() <reference-twig-function-render>` and `render_esi() <render_esi>`.
+:ref:`render() <reference-twig-function-render>` and :ref:`render_esi() <render_esi>`.
```

Comment:
You will have to add a label before the "render_esi" headline like this:

```rst
.. _reference-twig-function-render-esi:
```

and then change the referenced label name here accordingly.